manageiq-providers-vmware.gemspec

```diff
@@ -1,24 +1,28 @@
-$:.push File.expand_path("../lib", __FILE__)
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'manageiq/providers/vmware/version'
 
-require "manageiq/providers/vmware/version"
+Gem::Specification.new do |spec|
+  spec.name          = "manageiq-providers-vmware"
+  spec.version       = ManageIQ::Providers::Vmware::VERSION
+  spec.authors       = ["ManageIQ Authors"]
 
-Gem::Specification.new do |s|
-  s.name        = "manageiq-providers-vmware"
-  s.version     = ManageIQ::Providers::Vmware::VERSION
-  s.authors     = ["ManageIQ Developers"]
-  s.homepage    = "https://github.com/ManageIQ/manageiq-providers-vmware"
-  s.summary     = "Vmware Provider for ManageIQ"
-  s.description = "Vmware Provider for ManageIQ"
-  s.licenses    = ["Apache-2.0"]
+  spec.summary       = "ManageIQ plugin for the VMware provider."
+  spec.description   = "ManageIQ plugin for the VMware provider."
+  spec.homepage      = "https://github.com/ManageIQ/manageiq-providers-vmware"
+  spec.license       = "Apache-2.0"
 
-  s.files = Dir["{app,config,lib}/**/*"]
+  spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
+  spec.bindir        = "exe"
+  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  spec.require_paths = ["lib"]
 
-  s.add_dependency("fog-vcloud-director", ["~> 0.3.0"])
-  s.add_dependency "ffi-vix_disk_lib",        "~>1.1"
-  s.add_dependency "rbvmomi",                 "~>3.0"
-  s.add_dependency "vmware_web_service",      "~>2.0"
-  s.add_dependency "vsphere-automation-sdk",  "~>0.4.7"
+  s.add_dependency "fog-vcloud-director",    "~> 0.3.0"
```

Comment:
These should be `spec.add_dependency`

Reply:
Weird...I have that locally...must have not pushed it